components/ui/DownArrow.jsx

```diff
@@ -0,0 +1,9 @@
+export default function DownArrow(props){
```

Comment:
you can try to change DownArrow component to a descriptive name like <ArrowDown />